src/PlayerState.tsx

```diff
@@ -23,7 +23,7 @@ function PlayerState({
       <Row className="justify-content-sm-center my-3">
         <Col className="d-flex" sm="auto">
           {letters.map((letter, i) => (
-            <LetterCell key={i} letter={letter} />
+            <LetterCell key={i} letter={letter} selectPlayer={() => {return null}}/>
```

Comment:
rather than passing a null function, it's probably better to make the param optional

Reply:
done